ajax/libs/jquery.appear/package.json

```diff
@@ -16,7 +16,19 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/morr/jquery.appear"
+    "url": "https://github.com/morr/jquery.appear.git"
```

Comment:
Please don't update repo url in commit with commit msg `Update jquery.appear package.json with autoupdate property`. People always use commit msg to describe what they have done in a commit. So if you want, you should use another commit to do it. But here, the original is fine (no need to modify).